forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -32,56 +33,36 @@ internal class BTPinCollector(
         return vaultType
     }
 
-    override suspend fun collectPinForBalanceCheck(
+    override suspend fun submitBalanceCheck(
```

Comment:
Oh nice, so we're changing the name here? Yeah `collectPinFor...` always felt a bit jank to me. Like that we're finally doing this